elastalert/config.py

```diff
@@ -244,7 +244,8 @@ def _dt_to_ts_with_format(dt):
     if 'compound_aggregation_key' in rule:
         include += rule['compound_aggregation_key']
     if 'compare_key' in rule:
-        include.append(rule['compare_key'])
+        for val in rule['compare_key'].split(','):
```

Comment:
Since yaml supports lists, can you just make this field be either a single string or a list, and adjust the code accordingly? Similar to how query_key is handled.